docs/metrics.md

```diff
@@ -42,6 +41,29 @@ When we aggregate Lead time for changes we use the maximum of a period of time.
 | High                  | Between 1 day and 1 week     |
 | Elite                 | Less than 1 day              |
 
+### Sub-metrics
+
+There are several metrics that directly influence the lead time for changes,
+and can help you track in more detail what needs to be improved in your workflow.
+All the sub-metrics are aggregated by an interval (day, week or month) and have values for
+the average, median, and percentiles 5%, 20%, 25%, 75%, 80%, 95%.
+
+These metrics are extracted from pull requests,
+only include pull requests that have been merged (ignores closed without merge and currently open) and
+draft pull requests are treated as normal pull requests.
+
+#### Time to Open
```

Comment:
maybe a table makes more sense for this type of information (kind of reference)

Reply:
hmm, not sure what you mean

Reply:
something like 👇 
 metric name | metric description| metric formula | units 
-|-|-|-
time to open | ...
time in review | ...

Reply:
we can try that if we think it looks better